### DISCUSSION TITLE
How do I get the pinned tabs to have a colored background? (MacOS)

### DISCUSSION THREAD

**Discussion by snarkyalyx** [artsyfriedchicken/EdgyArc-fr#19] [General]
As shown in the image, I have been unable to reproduce (MacOS)

![CleanShot 2024-03-28 at 00 27 41](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/57874862/de93a990-8943-4b90-9913-259d9e537d5c)

Just looks like this for me:
![CleanShot 2024-03-28 at 00 28 42](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/57874862/9be1e772-1816-4376-8d02-e97a6bc94d07)

**Comment by snarkyalyx** [comment]
lmao found it, for any user in the future wondering, right-click the pinned tab and then set color - I am used to Arc doing this automatically: 
![CleanShot 2024-03-28 at 00 30 52](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/57874862/bb759fca-4461-44be-94d6-7fe278239d2c)

**Comment by artsyfriedchicken** [comment]
Yup yup! Ive also added a default background colour to it in my latest commit.
